docs/content/reference/routing-configuration/other-providers/swarm.md

```diff
@@ -0,0 +1,652 @@
+
+        Setting the label `traefik.http.services.xxx.loadbalancer.server.port`
+        overrides this behavior.
+
+??? example "Specifying more than one router and service per container"
+
+    Forwarding requests to more than one port on a container requires referencing the service loadbalancer port definition using the service parameter on the router.
+
+    In this example, requests are forwarded for `http://example-a.com` to `http://<private IP of container>:8000` in addition to `http://example-b.com` forwarding to `http://<private IP of container>:9000`:
+
+    ```yaml
+    version: "3"
+    services:
+      my-container:
+        # ...
+        deploy:
+          labels:
+            - traefik.http.routers.www-router.rule=Host(`example-a.com`)
+            - traefik.http.routers.www-router.service=www-service
+            - traefik.http.services.www-service.loadbalancer.server.port=8000
+            - traefik.http.routers.admin-router.rule=Host(`example-b.com`)
+            - traefik.http.routers.admin-router.service=admin-service
+            - traefik.http.services.admin-service.loadbalancer.server.port=9000
+    ```
+
+## Routing Configuration
+
+!!! info "Labels"
+
+     Labels are case-insensitive.
+
+### General
+
+Traefik creates, for each container, a corresponding [service](../http/load-balancing/service.md) and [router](../http/router/rules-and-priority.md).
+
+The Service automatically gets a server per instance of the container,
+and the router automatically gets a rule defined by `defaultRule` (if no rule for it was defined in labels).
+
+#### Service definition
+
+--8<-- "content/routing/providers/service-by-label.md"
+
+??? example "Automatic service assignment with labels"
+
+    With labels in a compose file
+
+    ```yaml
+    labels:
+      - "traefik.http.routers.myproxy.rule=Host(`example.net`)"
+      # service myservice gets automatically assigned to router myproxy
+      - "traefik.http.services.myservice.loadbalancer.server.port=80"
+    ```
+
+??? example "Automatic service creation and assignment with labels"
+
+    With labels in a compose file
+
+    ```yaml
+    labels:
+      # no service specified or defined and yet one gets automatically created
+      # and assigned to router myproxy.
+      - "traefik.http.routers.myproxy.rule=Host(`example.net`)"
+    ```
+
+### Routers
+
+To update the configuration of the Router automatically attached to the container,
+add labels starting with `traefik.http.routers.<name-of-your-choice>.` and followed by the option you want to change.
+
+For example, to change the rule, you could add the label ```traefik.http.routers.my-container.rule=Host(`example.com`)```.
+
+!!! warning "The character `@` is not authorized in the router name `<router_name>`."
+
+??? info "`traefik.http.routers.<router_name>.rule`"
+
+    See [rule](../http/router/rules-and-priority.md) for more information.
+
+    ```yaml
+    - "traefik.http.routers.myrouter.rule=Host(`example.com`)"
+    ```
+
```

Comment:
traefik.http.routers.<router_name>.rulesyntax is missing
traefik.http.routers.<router_name>.observability.accesslogs is missing
traefik.http.routers.<router_name>.observability.tracing is missing
traefik.http.routers.<router_name>.observability.metrics is missing